spring-security-modules/spring-security-azuread/src/main/java/com/baeldung/security/azuread/config/JwtAuthorizationConfiguration.java

```diff
@@ -45,20 +39,18 @@ private OAuth2UserService<OidcUserRequest, OidcUser> customOidcUserService(JwtAu
           props.getGroupsClaim(),
           props.getGroupToAuthorities());
         
-        return (userRequest) -> {
+        return userRequest -> {
             OidcUser oidcUser = delegate.loadUser(userRequest);
             // Enrich standard authorities with groups 
             Set<GrantedAuthority> mappedAuthorities = new HashSet<>();
             mappedAuthorities.addAll(oidcUser.getAuthorities());
             mappedAuthorities.addAll(mapper.mapAuthorities(oidcUser));
-            
+
             oidcUser = new NamedOidcUser(mappedAuthorities, oidcUser.getIdToken(), oidcUser.getUserInfo(),oidcUser.getName());
 
             return oidcUser;
         };
     }
-
-    
     
 //    @Bean
```

Comment:
Do we know why we have commented code here?

Reply:
Like, is it mentioned in the article that this bean can also be used or something.

Reply:
Yes I noticed that, maybe we can remove the commented code as part of the upgrade, If it is not needed of course.
It can be that it is there for spting boot 2 but no longer relevant for spring boot 3.
Can you verify please if we need to retain it.

Reply:
Sure, I will check

Reply:
As a conclusion: the MappingJwtGrantedAuthoritiesMapper doesn't exist, so in case some one will uncomment that code is unusable.
Do you suggest to remove that pice of commented code?

Reply:
Yes please, it seems it is some previous code that was forgotten to be removed. Thanks!

Reply:
Done